docs/index.rst

```diff
@@ -294,3 +361,68 @@ A single page dialog appears that allows you to change any of the parameters of
+monitor the flow of data within a pipeline. We can use the statistics
+that the north service updates for the data processed by the north
+service to determine if the service is functioning and sending data out
+of the Fledge instance. In this case the name of the statistic we want
+to monitor is the same as the name of the service.
+
+We will assume we have a service called *ADH* that sends data to the AVEVA Data Hub. 
+
+  - We create a notification as described above, selecting the *Threshold* filter for the notification.
+
+    +-----------+
+    | |ADHRule| |
+    +-----------+
+
+  - We select the *Data Source* as *Statistics History*, this will gives a a value every 15 seconds of the number of readings sent to ADH in the previous 15 seconds. We expect at least one value will be sent every 15 seconds.
+
+  - Select a condition of *<* and enter a value of *1*.
+   
+This will cause the notification to trigger if the value of the statistic
+is less than 1. If we wanted to trigger on a low rather than 0 flow of
+data then we can obviously increase this value. Of course that is reliant
+on the user knowing what a reasonable value is. It might be better, if an
+alert is required when the flow drops of to use the 8Average* filter and
+define if the flow rate drop by 10%, or whatever percentage is required,
+below the observed average flow rate then raise a notification.
+
+If the flow rate is low, or we wanted to look at rates other than the 15
+seconds samples we can use the *Window* evaluation and set the period we
+would like to observe values over. The example below shows how we would
+look at the average flow rate per minute.
+
++--------------------+
+| |ADHRatePerMinute| |
++--------------------+
+
+We can configure any of the supported delivery plugins for our new
+notification dependent on how and who or what we would like to inform
+regarding this condition. It may be a simple message to an operator,
+raising a ticket in a ticketing system or issue some control message to
+cause fail over to a standby system.
+
+Audit Data Example
+------------------
+
+A simple example of a notification based on audit data would be to send a notification any time any service restarts. We do this by looking for particular audit codes, the audit codes that are supported by the system are defined in the |audit_trail| section.
+
+The audit code we will use in this case is SRVRG which is written whenever
+a new service instance registers with the Fledge core.
+
+As with any notification we start by creating a naming the notification. Then we select the notification rule plugin. In this case we will use the *Data Availability* plugin. This rule triggers every time it receives data that matches the asset name or audit name it is given.
+
+We enter the required audit code in the *Audit Code* field. If we wanted to monitor several audit codes we could a comma separated list, however in this case one entry is sufficient.
```

Comment:
If we wanted to monitor several audit codes we could a comma separated list

If we want to monitor several audit codes we could enter a comma separated list e.g. SRVRG, SRVUN,